src/aura/STG_CMP_Addr/STG_CMP_AddrHelper.js

```diff
@@ -0,0 +1,17 @@
+({
+	runBackfill : function (component, event, helper) {
+		// console.log();
+		var runBatchAction = component.get("c.executeEthnicityRaceBatch");
+		runBatchAction.setCallback(this, function(response) {
+	        if(response.getState() === "SUCCESS") {
+
+			    var backfillBtn = component.find("ethnicRaceMsg");
+			    $A.util.toggleClass(backfillBtn, "slds-hide");
+
+	        } else if(response.getState() === "ERROR") {
+	        	this.displayError(response);
```

Comment:
Does the displayError method still work properly? In the Advisee Appt component in SAL the error message is retrieved just with `errors[0].message`.